# HG changeset patch
# User Augie Fackler <augie@google.com>
# Date 1542214333 18000
#      Wed Nov 14 11:52:13 2018 -0500
# Branch stable
# Node ID 4790732559ade2c1ab13608ba8d9e0c22d335b9b
# Parent  e5f54c4ec075f28770ae16619a839e6431df0935
tests: allow for 100% of profiled time in sleep in test-profile.t

I'm getting an annoying failure in this test on our builder, and I
*think* what's happening is that the profiler is taking _just_ long
enough to start that we're spending 100% of the profiled time in the
sleep function, which was causing the leading space to not be printed
since the 100 was in the first column of output.

Differential Revision: https://phab.mercurial-scm.org/D5272

diff --git a/tests/test-profile.t b/tests/test-profile.t
--- a/tests/test-profile.t
+++ b/tests/test-profile.t
@@ -85,17 +85,17 @@ statistical profiler works
   $ cat ../out | statprofran
 
 Various statprof formatters work
 
   $ hg --profile --config profiling.statformat=byline sleep 2>../out
   $ head -n 3 ../out
     %   cumulative      self          
    time    seconds   seconds  name    
-   * sleepext.py:*:sleep (glob)
+  * sleepext.py:*:sleep (glob)
   $ cat ../out | statprofran
 
   $ hg --profile --config profiling.statformat=bymethod sleep 2>../out
   $ head -n 1 ../out
     %   cumulative      self          
   $ cat ../out | statprofran
 
   $ hg --profile --config profiling.statformat=hotpath sleep 2>../out
